projects/vdk-plugins/vdk-oracle/src/vdk/plugin/oracle/oracle_configuration.py

```diff
@@ -74,3 +78,8 @@ def add_definitions(config_builder: ConfigurationBuilder):
             default_value=None,
             description="The password secret key if using secrets to connect to Oracle",
         )
+        config_builder.add(
+            key=ORACLE_THICK_MODE,
+            default_value=True,
+            description="Use oracle thick mode, default is True. Set to False to disable oracle thick mode",
```

Comment:
I'd love a short summary of what thick mode is since it seems to be something specific to oracledb and I had to google it.